.testcaferc-parity.js

```diff
@@ -13,4 +13,5 @@ module.exports = {
   userVariables: {
     v3: true,
   },
+  assertionTimeout: 10000,
```

Comment:
@shawyu-okta bacon failed 8 times on the factor beacon assertion.  I increased the assertionTimeout here and it passed.  What do you think of temporarily keeping this in to make it less flaky until we come up with a better solution?